app/celery/scheduled_tasks.py

```diff
@@ -189,3 +199,140 @@ def check_templated_letter_state():
                 message=msg,
                 ticket_type=zendesk_client.TYPE_INCIDENT
             )
+
+
+def _get_dynamodb_comp_pen_messages(table, message_limit: int) -> list:
+    """
+    Helper function to get the Comp and Pen data from our dynamodb cache table.
+
+    :param table: the dynamodb table to grab the data from
+    :param message_limit: the number of rows to search at a time and the max number of items that should be returned
+    :return: a list of entries from the table that have not been processed yet
+    """
+
+    results = table.scan(
+        FilterExpression=boto3.dynamodb.conditions.Attr('is_processed').eq(False),
+        Limit=message_limit
+    )
+
+    items: list = results.get('Items')
```

Comment:
This will cause an error in the While loop because it returns None as default (can't do `len(None)`). This should critically end if it returns None.

Reply:
Scan always returns `Items` even if it's an empty list. I'll check for that case, but I don't think it's totally necessary here unless there's an issue with the scan statement itself, which would throw an exception and be caught in the calling function anyway.